pkg/config/config.go

```diff
@@ -79,7 +79,18 @@ func (c *Config) LoadData() error {
 		c.DataLoader = LoadFromEnvironment()
 	}
 
-	return c.DataLoader(c)
+	if err := c.DataLoader(c); err != nil {
+		return err
+	}
+
+	if c.Debug {
+		ns := "(global)"
```

Comment:
I was trying to be clear in the debug statement what namespace we were using even when the string was empty (since printing the empty string looked wrong). I've changed it to print `''` to be consistent.